packages/bpk-component-scrollable-calendar/stories.js

```diff
@@ -94,6 +96,38 @@ storiesOf('bpk-component-scrollable-calendar', module)
       maxDate={DateUtils.addMonths(new Date(), 12)}
     />
   ))
+  .add('Scrollable Calendar in a tall container', () => (
+    <div style={{ height: '500px', display: 'flex' }}>
+      <ScrollableCal
+        weekStartsOn={1}
+        daysOfWeek={weekDays}
+        formatMonth={formatMonth}
+        formatDateFull={formatDateFull}
+        DateComponent={BpkScrollableCalendarDate}
+        showWeekendSeparator
+        selectTodaysDate
+        // Subtract one day from today's date to make today selectable by default
+        minDate={DateUtils.addDays(new Date(), -1)}
+        maxDate={DateUtils.addMonths(new Date(), 12)}
+      />
+    </div>
+  ))
+  .add('Scrollable Calendar with a single month', () => (
```

Comment:
Whats the motivation for this story?

Reply:
To show that if there's more space than the component needs, it will align to the top of the space.